### PR TITLE
Force some words to translate in other languages

### PR DESCRIPTION
**Pull request by tassoevan**
Closes #10920

### CODE REVIEW

File: packages/rocketchat-ui-flextab/client/tabs/userInfo.html [hunk 1]
```diff
@@ -60,16 +60,10 @@ <h3 title="{{name}}" class="rc-user-info__name"><i class="status-{{status}}"></i
 							{{#if hasPermission 'view-full-other-user-info'}}
 								{{#if hasEmails}}
 									<div class="rc-user-info-details__item">
-										<label class="rc-user-info-details__label	">Email address</label>
+										<label class="rc-user-info-details__label	">{{_ "Email"}}</label>
 										{{#each emails}} <a href="mailto:{{address}}" class="rc-user-info-details__info">{{address}}{{#if verified}}&nbsp;<i class="icon-ok success-color"></i>{{/if}}</a> {{/each}}
 									</div>
 								{{/if}}
-								{{#if hasPhone}}
```
Comment (rodrigok) [inline packages/rocketchat-ui-flextab/client/tabs/userInfo.html]: Why did you remove the phone field?

Comment (tassoevan) [reply packages/rocketchat-ui-flextab/client/tabs/userInfo.html]: I brought it back, but I'm not sure how to add it as a custom field to test it.

Comment (rodrigok) [reply packages/rocketchat-ui-flextab/client/tabs/userInfo.html]: It's a livechat thing
